apps/explorer/src/components/GasBreakdown/index.tsx

```diff
@@ -91,8 +94,15 @@ function TotalGasAmount({ amount }: GasProps) {
 }
 
 function GasPaymentLinks({ objectIds }: { objectIds: string[] }) {
+    const isScrollable = objectIds.length > MAX_GAS_PAYMENT_LENGTH;
+
     return (
-        <div className="flex flex-wrap items-center gap-x-4">
+        <div
+            className={clsx(
+                'flex flex-wrap items-center gap-x-4 gap-y-2',
+                isScrollable && 'h-20 overflow-y-scroll'
```

Comment:
Why not just do max-h-20 and overflow scroll? Then you wouldn't need this logic?

Reply:
Reply